### ISSUE TITLE
Documentation: Can the changelog link to the GitHub issue?

### ISSUE THREAD

**Issue by Mariatta** [aio-libs/aiohttp#2888]
## Long story short

In the [changelog](http://aiohttp.readthedocs.io/en/stable/changes.html#aiohttp-changes), I see references to GitHub issue numbers.
For example, the changelog entry for 3.1.0:
```
3.1.1 (2018-03-27)
Support asynchronous iterators (and asynchronous generators as well) in both
client and server API as request / response BODY payloads. (#2802)
```

I suspect `#2802` refers to GitHub issue number #2802. Would be nice if that is linked.
 
## Expected behaviour

`#2802` links to #2802 so readers can easily look up the issue.

## Actual behaviour

`#2802` was just a plain text.
 
Thanks.

**Comment by asvetlov** [comment]
I like the idea.
Can be done easily.
I'll try to fix the changelog on making the next release.

**Comment by asvetlov** [comment]
Unfortunately, the problem is a little more complex than I expected.
I thought it can be solved by simple regex but #xxxx can point either on issue or on pull request.
Need to use github API for resolving.
Will do it soon anyway, just postponed yesterday to not block the release.

**Comment by samuelcolvin** [comment]
Simple fix: just link to `https://github.com/aio-libs/aiohttp/pull/XXX` and it will redirect if that number is an issue. eg. `https://github.com/aio-libs/aiohttp/pull/288` will redirect to this issue.

**Comment by samuelcolvin** [comment]
see [here](https://github.com/samuelcolvin/pydantic/blob/master/docs/conf.py#L25-L33) for how I do it in pydantic, I'm sure there's a more sophisticated way of doing it but that works.

**Comment by asvetlov** [comment]
Fixed by 96e09c6a
@samuelcolvin thanks for the idea
